### PR TITLE
Adding notecards and rooms documents to static/documents directory ...

### PR DESCRIPTION
**Pull request by fghalasz**
so that they can easily be accessed from online.interlisp.org.

### CODE REVIEW

File: static/documentation/ROOMSTECHDESC-NOTE.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The ROOMSTECHDESC.pdf document in tis directory was sourced from Interlisp.org Google Drive repository file
+medley-pdfs/rooms-pdfs/xd0e-RELEASE-rooms-2.0-users-src-ROOMSTECHDESC.PDF.
+
+FGH 2022-12-13
+
```
Comment (masinter) [inline static/documentation/ROOMSTECHDESC-NOTE.txt:5]: should we have a loadup/build task of making these PDFs from their (TEdit? Notecards?) sources?

Comment (fghalasz) [reply static/documentation/ROOMSTECHDESC-NOTE.txt:5]: For now, the Notecards doc is a scanned copy.  Haven't been able to recreate the right Notecards manual from TEdits.  Working on it,

Also for now, the ROOMS doc is a placeholder - this is not really the right doc for a ROOMS User Guide and/or primer.

Once we get legitimate TEdit-based docs straightened out, we should probably add the pdf generation to loadups.  But for now, probably just storing these docs in a static folder is the right thing to do.

**Review by masinter — APPROVED** [review]
please add (possibly low-priority) issues for the remaining doc work
including automating making PDFs from TEdit.

**Comment by masinter** [comment]
@fghalasz : @Anzus has some documents from his attic that might be helpful. Did you see the email?

**Pull request merged**